### PR TITLE
Added formatter for elasticsearch

### PR DESCRIPTION
**Pull request by pielco11**

### CODE REVIEW

File: elasticsearch/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## 0x00
+`sudo -H pip3 install elasticsearch`.
+
+## 0x01
+Download Elasticsearch and Kibana (works with 6.0.0, going to try with newest) from [here](https://www.elastic.co/downloads/past-releases)
```
Comment (Lonami) [inline elasticsearch/README.md]: "going to try with newest" is not something I would like to see in a proper documentation file like this.

File: elasticsearch/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+## 0x02
+If you downloaded the zip version, unzip `elasticsearch` and move to that `/bin` subdir. Run `./elasticsearch`
+The same for `kibana`.
+You should not see errors, if so: ask us.
+
+## 0x03
+Go to `http://localhost:5200`, `Dev Tools`, copy&paste from `index.json` and click the `green arrow`.
+Go to `Management`, `Index Patterns`, `Create Index Pattern` and digit `tgexport2`. **Don't choose time field**.
+
+## 0x04
```
Comment (Lonami) [inline elasticsearch/README.md]: These custom titles seem somewhat unnecessary. You could nest a list like this:
1. Step one.
2. Step two.
 2.1. Substep.
 2.2. Substep.
3. Step three.

File: formatters/esformatter.py [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+import sys
+
+from elasticsearch import Elasticsearch, helpers
+
+path_to_db = sys.argv[1]
+
+es = Elasticsearch()
+connect = sqlite3.connect(path_to_db)
+
+def push_messages():
```
Comment (Lonami) [inline formatters/esformatter.py]: This should subclass the `BaseFormatter` class, if it's going to be under `formatters/`.

**Comment by Lonami** [comment]
Looks good but there's still some work to do. Also please try sticking to a 80 characters column, like everywhere else in the project. I assume it's a work in progress, so there's no need to merge right away.

File: requirements.txt [hunk 4]
```diff
@@ -1,2 +1,3 @@
 telethon-aio
 tqdm
+elasticsearch
```
Comment (expectocode) [inline requirements.txt]: As this is optional, I wouldn't put it here.

File: formatters/esformatter.py [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
+import sys
+import tarfile
+import urllib.request
+import subprocess
+
+from elasticsearch import Elasticsearch, helpers
+
+from formatters import BaseFormatter
+
+class RecicleObject(object):
```
Comment (expectocode) [inline formatters/esformatter.py]: 'Recycle'

**Comment by Aleyasen** [comment]
@pielco11 @expectocode What happened to this PR? Any plan to merge it?

**Pull request closed without merging**